pulser-core/pulser/result.py

```diff
@@ -95,13 +95,15 @@ def get_samples(self, n_samples: int) -> Counter[str]:
         Returns:
             Samples of bitstrings corresponding to measured quantum states.
         """
-        dist = np.random.multinomial(n_samples, self._weights())
```

Comment:
This is being used in `QutipState.sample()` too, you might want to change it there as well

Reply:
I'm still thinking how to do this